roles/pmk-selinux/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- copy:
+    src: "{{ role_path }}/files/pf9.te"
+    dest: /tmp/pf9.te
+- name: pmk-selinux
+  shell: |
```

Comment:
Try using the `yum` module to install packages. Also, avoid use of the `shell` module unless you need to use globbing, pipes, etc.

```yaml
- name: Install SELinux packages
  yum:
    state: present
    name: "{{ item }}"
  with_items:
    - checkpolicy
    - policycoreutils
    - policycoreutils-python

- name: Load SELinux policies
  command: "{{ item }}"
  args:
    chdir: /tmp
  with_items:
    - checkmodule -M -m -o pf9.mod pf9.te
    - semodule_package -o pf9.pp -m pf9.mod
    - semodule -i pf9.pp

Reply:
thanks ! done , pushing this up now